src/listpack.c

```diff
@@ -670,8 +672,11 @@ unsigned char *lpFind(unsigned char *lp, unsigned char *p, unsigned char *s,
             skipcnt--;
         }
 
-        /* Move to next entry */
-        p = lpNext(lp, p);
+        /* Move to next entry, avoid use `lpNext` due to `ASSERT_INTEGRITY` in
+         * `lpNext` will call `lpBytes`, will cause performance degradation */
+        p = lpSkip(p);
+        assert(p >= (lp + LP_HDR_SIZE) && p < (lp + lp_bytes));
+        if (p[0] == LP_EOF) break;
```

Comment:
it would be a good idea to run the corrupt-dump-fuzzer with `--accurate` (or even edit the test and change the limits to run for a several hours).
and then repeat that with valgrind.
maybe we can do that later, after converting more data types to listpack.

Reply:
I wull do it tomorrow.

Reply:
i understand you followed that advise recently?
i.e. change `min_duration` or `min_cycles` to very long, and run with a long `--timeout` for several hours?

Reply:
@oranagra It's been running for 20 hours, no problems were found.

Reply:
@oranagra Sorry for my mistake, I found the issue from corrupt-dump-fuzzer in the process of making changes.
I don't know why it didn't come out the last time I ran it, silly me.
I will carry out one more.